src/system-script.js

```diff
@@ -46,7 +55,7 @@ export function fetch(load) {
   // and resolve with an empty string. The proper module resolution
   // happens inside the instantiate hook
   return new Promise((resolve, reject) => {
-    const name = normalizeName(load.name.substring(window.location.origin.length + 1));
```

Comment:
This is confusing, but SystemJS does some weird stuff with the `load.name`. When you are on a domain like: `app.canopytax.com/#some-path` the `load.name` is just `https://app.canopytax.com/canopy-styleguide` but when you are on a domain like: `app.canopytax.com/some/path#some-hash` then the load is `https://app.canopytax.com/some/canopy-styleguide` and when you are on `app.canopytax.com/some/path/#some-hash` it is `https://app.canopytax.com/some/path/canopy-styleguide`. The point of this logic is to try and parse just `canopy-styleguide`. It might be easier if I just use a regex.